modules/misc/tmpfiles.nix

```diff
@@ -37,7 +37,7 @@ in
           # Please change the option ‘systemd.user.tmpfiles.rules’ instead.
           ${lib.concatStringsSep "\n" cfg.rules}
         '';
-        onChange = "${pkgs.systemd}/bin/systemd-tmpfiles --user --create";
+        onChange = "${pkgs.systemd}/bin/systemd-tmpfiles --user --remove --create";
```

Comment:
Should we add `--clean` while we're here? I'm not too familiar with systemd-tmpfiles but reading the documentation mentions that as another operation.